topics/sequence-analysis/tutorials/quality-control/tutorial.md

```diff
@@ -22,6 +22,7 @@ follow_up_training:
     tutorials:
       - mapping
 time_estimation: "1H"
```

Comment:
is this still realistic with all the extra content?

Reply:
Maybe not...